src/app/assessment/assessment.service.ts

```diff
@@ -161,12 +163,20 @@ export class AssessmentService {
                 name: questionChoice.AssessmentChoice.name,
                 explanation: this.utils.has(questionChoice, 'AssessmentChoice.explanation') ? questionChoice.AssessmentChoice.explanation : ''
               });
+              if (this.utils.has(questionChoice, 'AssessmentChoice.description') && questionChoice.AssessmentChoice.description) {
+                info += "<p>" + questionChoice.AssessmentChoice.name + " - " + questionChoice.AssessmentChoice.description + "</p>";
+              }
             });
+            if (info) {
+              // Add the title
+              info = '<h3>Choice Description:</h3>' + info;
```

Comment:
@shawnm0705 just a note, Angular has [directive](https://angular.io/guide/dynamic-component-loader) and [pipe](https://angular.io/guide/pipes) to handle the template processing stuff. We can further enhance the code next time.